completions/chflags

```diff
@@ -0,0 +1,31 @@
+# chflags(1) completion                                      -*- shell-script -*-
+
+_comp_cmd_chflags()
+{
+    local cur prev words cword comp_args
+    _comp_initialize -- "$@" || return
+
+    if [[ $cur == -* ]]; then
+        # Complete -options
+        local w opts=""
+        for w in "${words[@]}"; do
+            [[ $w == -R ]] && opts="-H -L -P" && break
+        done
+        _comp_compgen -- -W '-f -h -v -R $opts'
+    else
+        local ret
```

Comment:
Per #987
```suggestion
        local REPLY
```

Reply:
@Low-power Thanks, I confirmed this is fixed in your last push.